### DISCUSSION TITLE
Attempting to use cfclient on Apple M2 chip

### DISCUSSION THREAD

**Discussion by figgythenewton** [bitcraze/discussions#1012] [Q&A] [answered]
My MacBook Air with an Apple M2 chip running Ventura 13.4 does not have support for the VirtualBox, so I am attempting to install and use the cfclient without the crazyflie VM. When installing python 3.9, I referred to the section under "Apple M1" in "Installation Instructions" on the crazyflie website to install brew x86_64 and python for my Mac. However, when I attempt to run the cfclient with "$ python3 -m cfclient.gui" I get this import error:

" ImportError: dlopen(/Users/lo359/Library/Python/3.9/lib/python/site-packages/PyQt5/QtCore.abi3.so, 0x0002): Symbol not found: __ZTVNSt3__13pmr25monotonic_buffer_resourceE
Referenced from: <645DDC2C-F655-324A-BE87-40804F3AC471> /Users/lo359/Library/Python/3.9/lib/python/site-packages/PyQt5/Qt5/lib/QtCore.framework/Versions/5/QtCore
  Expected in:     <54E8FBE1-DF0D-33A2-B8FA-356565C12929> /usr/lib/libc++.1.dylib "

I am not sure what to do. Any help is greatly appreciated!

**Comment by knmcguire** [comment, marked as answer]
Hi!

Seems to be a QT5 library dependency that is missing...

Since we are working on a release now, could you try one thing for us with a new QT library? We just have a release candidate out.

These are instructions for ubuntu but could you try to recreate it for your mac?

```
pip3 uninstall cflib cfclient
pip3 install cfclient==2023.10rc1
```

**Reply by figgythenewton** [reply]
This worked for me, thank you so much!

**Reply by knmcguire** [reply]
ahhh oke! well at least it works for you. Then it seemed to be a QT5 issue as the release candidate it has been upgraded to QT6. 

QT5 'should' work on m1's an m2's but there have been reports that it doesn't... so probably best that we are upgrading now.
